portfolio/src/main/java/com/google/sps/data/Keys.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+import com.google.gson.Gson;
+
+/*
+ * Store Keys used in servlets.
+ */
+public final class Keys {
+  public static final Gson GSON = new Gson();
```

Comment:
nit: GSON is not really a key. I think keeping GSON in DataServlet.java was fine